nuxt-app/components/ProfileCreationMainInfos.vue

```diff
@@ -47,10 +45,7 @@ function updatePicture(file: File, previewUrl: string) {
 
         <div class="intro-text mb-2 mt-5 text-base font-light text-white md:text-4xl" v-html="introText"></div>
         <div class="flex w-full flex-col items-center justify-center my-10">
-          <ProfilePicture class="h-24 w-24 md:h-64 md:w-64"
-                          :editable='true'
```

Comment:
Nur ne kleine Info, es geht ja jetzt raus, aber damit du es mal für die Zukunft gehört hast: 

Wenn man bools als prop fest auf `true` setzen möchte, kann man in Vue auch ganz easy nur `:editable` schreiben. Kleiner syntaktischer Zucker 😎